<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkClass</class>
 <widget class="QMainWindow" name="checkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>水样监测显示系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/boat_783px_1224913_easyicon.net.png</normaloff>img/boat_783px_1224913_easyicon.net.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>检测</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>连接</string>
             </property>
             <property name="icon">
              <iconset resource="check.qrc">
               <normaloff>:/check/img/down2_blue.png</normaloff>:/check/img/down2_blue.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>断开</string>
             </property>
             <property name="icon">
              <iconset resource="check.qrc">
               <normaloff>:/check/img/close_red.png</normaloff>:/check/img/close_red.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>载入检测点</string>
             </property>
             <property name="icon">
              <iconset resource="check.qrc">
               <normaloff>:/check/img/right3_green.png</normaloff>:/check/img/right3_green.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>保存</string>
             </property>
             <property name="icon">
              <iconset resource="check.qrc">
               <normaloff>:/check/img/set_orange.png</normaloff>:/check/img/set_orange.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>历史记录</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>370</width>
            <height>0</height>
           </size>
          </property>
          <widget class="QWidget" name="page"/>
          <widget class="QWidget" name="page_2"/>
         </widget>
        </item>
        <item>
         <widget class="QWebEngineView" name="webEngineView">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="check.qrc"/>
 </resources>
 <connections/>
</ui>
